packages/media-sound/cava/cava-0.10.0.exheres-0

```diff
@@ -37,11 +37,20 @@ DEFAULT_SRC_CONFIGURE_OPTION_ENABLES=(
     "pulseaudio input-pulse"
     "portaudio input-portaudio"
     "sdl output-sdl"
+    "sdl output-sdl-glsl"
```

Comment:
This requires a `build:` dep on `sys-devel/autoconf-archive` as stated by configure.ac:

```
configure: You need to install the autoconf archives to check for opengl. Remember to re-run autogen.sh.
configure: INFO: building without sdl_glsl (opengl) support
```
and also checks for and links to libGL so a `build+run:` dep on `dev-libs/libglvnd` as well.

Reply:
Thanks for adding those, they should however also be behind the sdl? option requirement